packages/mineral-ui-tokens/scripts/formats/index.js

```diff
@@ -0,0 +1,91 @@
+const keys = require('../keys');
+
+const formatInput = ({ input, valueTemplate }) => {
+  return input
+    .get('props')
+    .map((prop) => {
+      let result = [];
+      if (prop.has('comment')) {
+        result.push(`// ${prop.get('comment')}`);
+      }
+      const k = prop.get('name');
+      const v = prop.get('value');
+      result.push(valueTemplate(k, v));
+      return result;
+    })
+    .flatten(1)
+    .toArray()
+    .join('\n');
+};
+
+module.exports = {
+  colorExport: (input) =>
+    [
+      'export default {',
+      formatInput({
+        input,
+        valueTemplate: (k, v) => {
+          k = `_${k.split('_')[1]}`;
+          return `  ${k}: ${v},`;
```

Comment:
Decided not to use the JSON object because it doesn't add much value (we still need to manipulate `key` &/or `value` for each property). We did abstract common output to a shared function.